broker/core/utils/src/HttpClient.js

```diff
@@ -273,6 +274,7 @@ class HttpClient {
           // or, something happened in setting up the request that triggered an Error.
           const err = _.pick(error.toJSON(), [
             'message', 'name', 'description',
+            'config.url', 'config.method',
             'stack', 'code'
           ]);
           logger.error('HTTP request failed:', err);
```

Comment:
It would be better if we can combine the logger statements at L267 and L280. Something like:
`HTTP request failed for ${error.config.url}, HTTP Method: ${error.config.method} : err`